src/lib/playground-ide.ts

```diff
@@ -14,17 +14,17 @@
 
 import {LitElement, html, customElement, css, property} from 'lit-element';
 
-import './code-sample-project.js';
-import './code-sample-editor.js';
-import './code-sample-preview.js';
+import './playground-project.js';
+import './playground-file-editor.js';
+import './playground-preview.js';
 
 /**
  * A multi-file code editor component with live preview that works without a
  * server.
  *
- * <code-sample> loads a project configuration file and the set of source files
+ * <playground> loads a project configuration file and the set of source files
```

Comment:
Change to `<playground-ide>` here and 4 places below

Reply:
Oops, done.